src/components/Notification/StatusNotification.jsx

```diff
@@ -19,14 +21,22 @@ import StatusMessage from './StatusMessage';
  */
 
 const StatusNotification = ({ notification, statusType, defaultMessage, locationUrl = '' }) => (
-  <dl className="display">
-    <dt>{statusType}:</dt>
+  <Typography
```

Comment:
Line 24 `<Typogrphy>` component has a `<div>` inside it (Line 37). This will trigger an error in the console log. Switch this to perhaps a `<Box>`.